src/vite-env.d.ts

```diff
@@ -1 +1,2 @@
 /// <reference types="vite/client" />
+declare var config: any;
```

Comment:
```suggestion
// Remove duplicate reference to "vite/client" and add a type to the "config" variable

/// <reference types="vite/client" />


declare const config: Record<string, any>;
```

_Generated using [OpenSauced](https://opensauced.ai/)._

Reply:
Thank you for that.